app/controllers/application_controller.rb

```diff
@@ -1,4 +1,10 @@
 class ApplicationController < ActionController::Base
+  include Pundit
+
+  after_action :verify_authorized
+  after_action :verify_policy_scoped
```

Comment:
might want to add `unless: :devise_controller?` to skip `verify_authorized` and `verify_policy_scoped` for devise

Currently, I'm not able to load the live preview because of this

![Screenshot 2024-02-23 at 5 11 29 PM](https://github.com/WCW789/industrial-auth-1/assets/100433760/f6bbff32-5b0c-49f2-b712-39d6daa42486)